native_lib/build-cmdline.sh

```diff
@@ -0,0 +1,16 @@
+g++ -I/opt/homebrew/include/ -L/opt/homebrew/lib/ test_user_setup.c DDGSyncCrypto.c -lsodium -o test_user_setup
+
+# Smoke test
```

Comment:
are you planning to add run this in a GitHub Action workflow? I think we should.

Reply:
done

Reply:
for the record, you could have added that workflow yml on the same PR, instead of pushing straight to `main`.